main/keychain/src/main/java/org/cryptomator/keychain/LinuxSecretServiceAccess.java

```diff
@@ -0,0 +1,59 @@
+package org.cryptomator.keychain;
+
+import org.apache.commons.lang3.SystemUtils;
+import org.freedesktop.secret.simple.SimpleCollection;
+
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LinuxSecretServiceAccess implements KeychainAccessStrategy {
+	private SimpleCollection gnomeLoginKeyring = null;
+
+	@Inject
+	public LinuxSecretServiceAccess() {
+		try {
```

Comment:
```java
		SimpleCollection keyring = null;
		try {
			keyring = new SimpleCollection();
		} catch (Exception e) {
			// Accessing secret-service DBus API failed
		} finally {
			gnomeLoginKeyring = Optional.ofNullable(keyring);
		}
```

Reply:
Also, maybe log a warning in case of an exception?

Reply:
The messaging component of the secret-service lib logs an info for every request for the keyring. So I think there is no need for a further warning.